_posts/2022-10-03-the-definitive-guide-2.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Refresh of Trino: The Definitive Guide"
```

Comment:
Maybe "Release of the second edition of Trino: The Definitive Guide".

Mentioning the second edition of something feels exciting. Refresh feels like we're just fixing errata.